modules/reactor/src/meshgenerators/ReactorGeometryMeshBuilderBase.C

```diff
@@ -42,6 +43,12 @@ ReactorGeometryMeshBuilderBase::initializeReactorMeshParams(const std::string re
   declareMeshProperty("reactor_params_name", std::string(_reactor_params));
 }
 
+void
+ReactorGeometryMeshBuilderBase::freeReactorMeshParams()
+{
+  _reactor_params_mesh->reset();
```

Comment:
can we do this somewhere else?
like a finalize() ?

maybe at destruction ?

Reply:
Nope. Destruction is well after `generate()`.

Reply:
but we dont need to do this asap either

Reply:
The other option is to make it a warning and disable the warning for the entire module... which I am vehemently against

Reply:
nah we dont want that.

Ideally, we catch that we need to reset this object in some sort of base class routine for RGMB MGs

Reply:
We ll live with it. It s not a big limitation, though it s not very C++-ey to have to remember to free your memory yourself